src/test/resources/testData/discrete_ground_water_aqts.csv

```diff
@@ -1,17 +1,17 @@
+12,45646129,"field_visit_identifier","USGS","U.S. Geological Survey","335504116544201","USGS-335504116544201","06","GW","Well","33.91763056000000","-116.9124500000000","North American Datum of 1983","1000","1000",null,null,"2011-11-16 11:43:00","PST","-8:00","72019","2011-11-16","19:43","322.00",null,,null,"USGS","The reported water-level measurement represents a static level","m","Date is accurate to the Minute","S",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4113.0"
+13,45646128,"field_visit_identifier","USGS","U.S. Geological Survey","335504116541501","USGS-335504116541501","06","GW","Well","33.91775278000000","-116.9050389000000","North American Datum of 1983","1420","1440",null,null,"1992-02-14 12:00:00","UTC"," 0:00","72019","1992-02-14",null,"293",null,,null,"USGS","The reported water-level measurement represents a static level","D","Date is accurate to the Day","O",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4114.0"
+14,45646128,"field_visit_identifier","USGS","U.S. Geological Survey","335504116541501","USGS-335504116541501","06","GW","Well","33.91775278000000","-116.9050389000000","North American Datum of 1983","1420","1440",null,null,"1992-03-27 12:00:00","UTC"," 0:00","72019","1992-03-27",null,"335",null,,null,"USGS","The reported water-level measurement represents a static level","D","Date is accurate to the Day","O",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4115.0"
+15,45646128,"field_visit_identifier","USGS","U.S. Geological Survey","335504116541501","USGS-335504116541501","06","GW","Well","33.91775278000000","-116.9050389000000","North American Datum of 1983","1420","1440",null,null,"1993-04-29 12:00:00","UTC"," 0:00","62610","1993-04-29",null,null,"2026",,"National Geodetic Vertical Datum of 1929","USGS","The reported water-level measurement represents a static level","D","Date is accurate to the Day","R",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4116.0"
+16,45646128,"field_visit_identifier","USGS","U.S. Geological Survey","335504116541501","USGS-335504116541501","06","GW","Well","33.91775278000000","-116.9050389000000","North American Datum of 1983","1420","1440",null,null,"1993-06-04 12:00:00","UTC"," 0:00","61055","1993-06-04",null,null,"2070",,"National Geodetic Vertical Datum of 1929","USGS","The reported water-level measurement represents a static level","D","Date is accurate to the Day","R",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4117.0"
```

Comment:
I removed the "measurement_source_code" & "measurement_source" columns, but the contents of the 'level_accuracy' column is just wrong, so I deleted the contents.  I think the 'level_accuracy_code' contents are incorrect as well...
Our current code never does any logic on those columns, so it really has no impact, its just really misleading to look at as sample data.